pandas/tests/series/test_operators.py

```diff
@@ -962,6 +962,36 @@ def test_timedelta64_ops_nat(self):
 
 
 class TestDatetimeSeriesArithmetic(object):
+    def test_sub_datetime64_not_ns(self):
+        # GH#7996
+        ser = Series(date_range('20130101', periods=3))
+        dt64 = np.datetime64('2013-01-01')
+        assert dt64.dtype == 'datetime64[D]'
+        res = ser - dt64
+        expected = pd.Series([Timedelta(days=0), Timedelta(days=1),
+                              Timedelta(days=2)])
+        tm.assert_series_equal(res, expected)
+
+        res = dt64 - ser
+        tm.assert_series_equal(res, -expected)
+
+        dti = pd.DatetimeIndex(ser)
+        res = dti - dt64
+        tm.assert_index_equal(res, pd.Index(expected))
+
+        res = dt64 - dti
+        tm.assert_index_equal(res, pd.Index(-expected))
+
+    @pytest.mark.xfail(reason='GH#7996 datetime64 units not converted to nano')
+    def test_frame_sub_datetime64_not_ns(self):
```

Comment:
do we have a separate issue for this? if not let's create one and reference it (as closing #7996 with this PR)

Reply:
Just opened #18874.